### PR TITLE
optionally directly define socket paths; option to disable and set a timeout for ipisp; add helper script for development and update workflows

### PR DESCRIPTION
**Pull request by herbetom**
This PR combines a bunch of minor improvements.

With them all beeing in some aspect related i decided top open this big PR instead of numerous small ones. The commits should explain what's going on with each change.

### CODE REVIEW

File: fastd-exporter.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ var (
 	configPathPattern = flag.String("config-path", "/etc/fastd/%s/fastd.conf", "Override fastd config path, %s will be replaced with the fastd instance name.")
 	webListenAddress  = flag.String("web.listen-address", ":9281", "Address on which to expose metrics and web interface.")
 	webMetricsPath    = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+	ipispEnable       = flag.Bool("ipisp.enable", true, "enable usage of ipisp lookup")
+	ipispTimeout      = flag.Int("ipisp.timeout", 300, "milliseconds to wait for ipisp lookup to finish")
```
Comment (blocktrron) [inline fastd-exporter.go]: `ip-asn-lookup.{opt}` is more appropriate

File: fastd-exporter.go [hunk 2]
```diff
@@ -342,12 +353,24 @@ func main() {
 	}
 
 	for i := 0; i < len(instances); i++ {
-		config, err := parseConfig(instances[i])
+		instance := strings.SplitN(instances[i], "=", 2)
```
Comment (blocktrron) [inline fastd-exporter.go]: Regex for input validation?

File: contrib/fastd-exporter-start.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# Array to hold the PIDs of the fastd instances
+fastd_pids=()
+
+# Stop the fastd instances when the script exits
+trap 'for pid in "${fastd_pids[@]}"; do kill $pid; done' EXIT
+
+# Start the fastd instances
+for instance in "${fastd_instances[@]}"
+do
+    fastd --config $SCRIPT_DIR/fastd/$instance/fastd.conf --status-socket $SCRIPT_DIR/fastd-$instance.sock 2>&1 | sed "s/^/[fastd $instance] /" &
```
Comment (blocktrron) [inline contrib/fastd-exporter-start.sh:17]: I don't think the stream output provides any additional value

Comment (herbetom) [reply contrib/fastd-exporter-start.sh:17]: i think it's quite helpful to determine wheter fastd was succesfully started and the connections where established as expected.

File: fastd-exporter.go (hunk 1 above)
Comment (herbetom) [reply fastd-exporter.go]: yes, i've changed that

File: fastd-exporter.go (hunk 2 above)
Comment (herbetom) [reply fastd-exporter.go]: i've changed it up and there is now input validation. Hoewer, i didn't cover everything that would've worked previously.

**Pull request merged**